react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+class Collapse extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { open: false };
+    this.togglePanel = this.togglePanel.bind(this);
+  }
+
+  togglePanel() {
+    this.setState({ open: !this.state.open });
+  }
+
+  render() {
+    return (
+      <div>
+        <div onClick={this.togglePanel} className='header'>
+          {this.props.title}</div>
+        {this.state.open ? (
+          <div className='content'>
+            {this.props.children}
+          </div>
+        ) : null}
+      </div>);
+  }
+}
+
+export default class Accordian extends React.Component {
+  render() {
+    return (<div>
```

Comment:
After passing in the topics data via props you should be able to access the topics data with `this.props.topics` you can then loop the array to create your jsx